bin/cdn-upload

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env node
+process.on('unhandledRejection', err => console.error(err))
+
+const fs = require('fs')
+const path = require('path')
+const semver = require('semver')
+const S3 = require('aws-sdk/clients/s3')
+const Cloudfront = require('aws-sdk/clients/cloudfront')
+
+// INPUT PARAMETERS
+const packageDir = process.cwd()
+const packageJson = require(packageDir + "/package.json")
+const packageName = packageJson.name
+const packageVersion = packageJson.version
+const bucketName = 'bugsnagcdn'
+const distributionId = 'E205JDPNKONLN7'
```

Comment:
Could these be added using environment variables?

Reply:
They're static values for each package, so I think we can set them automatically based on the package name

It makes the script not _entirely_ generic, but I think it's better to be safe against accidentally uploading something to the wrong bucket or invalidating the wrong cache

So we could do something like this:

```js
const packageConfigs = new Map([
  ['@bugsnag/browser', { distributionId: '...', bucketName: '...' }],
  ['@bugsnag/web-worker', { distributionId: '...', bucketName: '...' }],
])

const config = packageConfigs.get(packageName)

if (!config) {
  throw new Error(`Unknown package name: ${packageName}`)
}

const distributionId = config.distributionId
const bucketName = config.bucketName
```